index.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉